docs/reports/create-a-draft.md

```diff
@@ -0,0 +1,59 @@
+
+## Create a draft report
+After logging into the [administration area](https://reports.opensafely.org/admin/) click "Add" next to Reports:
```

Comment:
Will external users have the regular admin permissions here? Is there the danger they could change existing reports?

Reply:
They will.  I'm going to tighten the permissions slightly but it won't stop any user with admin access from editing an existing report.  Do we think that's going to be a danger with the pilot users?

Reply:
I think it'll be fine for the pilot phase , I can warn appropriately!